okhttp/src/jvmMain/kotlin/okhttp3/internal/-JvmPlatform.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package okhttp3.internal
+
+import java.util.concurrent.TimeUnit as JavaTimeUnit
```

Comment:
It seems a shame that kotlin doesn't handle this issue for us.  But if we want both platforms to have their natural time unit implementation, without doubling all methods, it probably means this.

We can also define expect values for the enum as well, I assume.

Reply:
I think we’re actually stuck doubling our methods.

As-is I don’t think callers can use this API in multiplatform code, unless they exclusively use `okhttp3.internal.TimeUnit` in their implementations.